internal/converter/internal/staticconvert/internal/build/catchpoint_exporter.go

```diff
@@ -0,0 +1,20 @@
+package build
```

Comment:
converters are used to convert a Grafana agent static mode config to an Alloy config but your converter is not available for the grafana agent static so you don't need to add a converter

Reply:
@wildum Im sorry, I might be misunderstanding here. Would this mean I have to drop all my files related to the static mode or just these files
<img width="383" alt="image" src="https://github.com/grafana/alloy/assets/38274348/54a267f9-a5ce-4ff7-b3a1-9357bedd77a9">

Reply:
yeah all the files that are in the converter dir

Reply:
Alright sounds good! I went ahead and removed those files.